packages/jest-preset-mc-app/package.json

```diff
@@ -39,7 +39,7 @@
     "jest-transform-graphql": "2.1.0",
     "jest-watch-master": "1.0.0",
     "jest-watch-typeahead": "0.4.0",
-    "make-plural": "5.0.0",
+    "make-plural": "6.0.1",
```

Comment:
The imports changed.

<img width="520" alt="CleanShot 2019-10-22 at 09 44 56@2x" src="https://user-images.githubusercontent.com/1877073/67266758-41182180-f4b1-11e9-8d09-cfdd7beb6340.png">